cirq/ops/common_gates.py

```diff
@@ -505,7 +505,8 @@ class CNotGate(eigen_gate.EigenGate,
                gate_features.CompositeGate,
                gate_features.TwoQubitGate,
                gate_features.QasmConvertableGate):
-    """A controlled-NOT. Toggle the second qubit when the first qubit is on."""
+    """A controlled-NOT. Toggle the second qubit when the first qubit is on.
+    You can also use the keyword arguments 'target' and 'control' instead of the positional ones."""
```

Comment:
This sentence is a bit out of place. I would go with something more like

"When applying CNOT to qubits, you can either use positional arguments `CNOT(q1, q2)`  or named arguments `CNOT(control=q1, target=q2)`. (Mixing the two is not permitted.)"